core/src/test/java/com/linecorp/armeria/common/logging/DefaultRequestLogTest.java

```diff
@@ -493,4 +493,19 @@ void toStringWithChildren() {
         assertThat(lines[2]).matches("^\\t\\{Request: \\{.*}, Response: \\{.*}}$");
         assertThat(lines[3]).matches("^\\t\\{Request: \\{.*}, Response: \\{.*}}$");
     }
+
+    @Test
+    void testGetIfAvailable(){
+        // Given
+        final ServiceRequestContext ctx = ServiceRequestContext.of(HttpRequest.of(HttpMethod.GET, "/"));
+        final DefaultRequestLog log = (DefaultRequestLog) ctx.log();
```

Comment:
Question) Is there a need to cast this?

```suggestion
        final RequestLogAccess log = ctx.log();
```